.gitpod.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM gitpod/workspace-full
+
+ENV PATH /home/linuxbrew/.linuxbrew/bin:$PATH
+
+# More information: https://www.gitpod.io/docs/config-docker/
+RUN sudo rm -rf /usr/bin/hd && \
+    brew install linuxsuren/linuxsuren/hd && \
+    hd install cli/cli
```

Comment:
It seems that github cli isn't installed correctly.

```bash
gitpod /home/linuxbrew/.linuxbrew $ whereis hd
hd: /usr/bin/hd /usr/share/man/man1/hd.1.gz
gitpod /home/linuxbrew/.linuxbrew $ whereis gh
gh:
```

Reply:
It works for me. You an visit [this link](http://gitpod.io/#/github.com/linuxsuren/devops-agent/tree/gitpod).

![image](https://user-images.githubusercontent.com/1450685/118749050-57ee4800-b88f-11eb-930d-bd5868dfda7b.png)

Reply:
Sorry for that. I went into <https://gitpod.io/#https://github.com/kubesphere/devops-agent> incorrectly.